src/components/field-radio.tsx

```diff
@@ -0,0 +1,49 @@
+import {
+  FieldValues,
+  useController,
+  UseControllerProps
+} from "react-hook-form";
+
+import Label from "@/components/box-label";
+import Error from "@/components/error-msg";
+
+export interface FormProps<T extends FieldValues = FieldValues>
+  extends UseControllerProps<T> {
+  label?: string;
+  placeholder?: string;
+}
+
+/*
+This is a component for the HTML `<radio>' tag
+Will display 2 radio buttons asking displaying 'Yes' or 'No'
+Input:
+  label: The label displayed on the text box
+Output:
+  A input radio option box for a 'yes' or 'no' question that is compatible w/ React-hook-forms 
+*/
+export default function Field_Radio<T extends FieldValues = FieldValues>({
+  label,
+  ...props
+}: FormProps<T>) {
+  const { field, fieldState } = useController(props);
+
+  !label ? (label = props.name) : "";
+
+  return (
+    <div className="relative mb-2 flex h-12 w-64 flex-row items-center justify-between rounded-lg border border-grey-300 px-3 shadow">
+      <Label label={label} {...props} />
+      <div className="m-auto flex flex-row items-start gap-1 text-xs">
+        <label htmlFor={`${props.name}-y`} className="mr-8 flex gap-1">
```

Comment:
You can remove the`mr-8`, and change the gap value in the parent
```suggestion
      <div className="m-auto flex flex-row items-start gap-4 text-xs">
        <label htmlFor={`${props.name}-y`} className="flex gap-1">
```

Reply:
yep, fixed